lib/modules/apostrophe-areas/public/js/user.js

```diff
@@ -323,6 +323,41 @@ apos.define('apostrophe-areas', {
       return editors;
     };
 
+    // Loads the virtual areas inside a modal.
+    self.loadVirtualArea = function () {
+      var $virtualAreas = $('[data-virtual-area]');
```

Comment:
No global jquery searches, should be scoped via $el.find() which should be an argument.

Reply:
Actually this whole thing should be a method of apostrophe-modal since that's where the action is. Then you can enable/disable the save operation neatly rather than adding and removing classes globally.